pages/common/btop.md

```diff
@@ -11,3 +11,11 @@
 - Start `btop` with the specified settings preset:
 
 `btop --preset {{0..9}}`
+
+- Start `btop` in tty mode using 16 colors and tty friendly graph symbols:
+
+`btop --tty_on`
+
+- Start `btop` with in 256-color mode instead of 24-bit color mode:
```

Comment:
```suggestion

- Start `btop` in TTY mode using 16 colors and TTY-friendly graph symbols:

`btop --tty_on`

- Start `btop` within 256-color mode instead of 24-bit color mode:
```